www/source/index.html.slim

```diff
@@ -12,7 +12,7 @@ title: InSpec - Audit and Test Framework
         infrastructure. You can also use it to verify if your servers and
         applications are configured correctly.
     p.text-center.home-hero--buttons
-      a.button.secondary href="" Download Inspec
+      a.button.secondary href="https://downloads.chef.io/inspec" Download Inspec
```

Comment:
I guess I didn't notice this line at all because it also says "Inspec" instead of "InSpec"